docs/rules/no-invalid-properties.md

```diff
@@ -44,9 +44,25 @@ body {
 }
 ```
 
-### Limitations
+## Options
 
-When a variable is used in a property value, such as `var(--my-color)`, the rule can only properly be validated if the parser has already encountered the `--my-color` custom property. For example, this will validate correctly:
+This rule accepts an option which is an object with the following property:
+
+- `allowUnknownVariables` (default: `false`) - Ignore variables that cannot be traced to custom properties in the current file
+
+When a variable is used in a property value, such as `var(--my-color)`, the rule can only properly be validated if the parser has already encountered the `--my-color` custom property. With `{ allowUnknownVariables: false }`, unknown variables will result in a lint error. With `{ allowUnknownVariables: true }`, the property value will be ignored and only the property name will be validated.
+
+Examples of **incorrect** code with `{ allowUnknownVariables: false }` (the default):
+
+```css
+a {
+	color: var(--my-color);
+}
+```
+
+This code uses `var(--my-color)` before `--my-color` was defined, or where it is defined in another CSS file. Therefore, `color: var(--my-color)` cannot be property validated.
+
+Examples of **correct** code with `{ allowUnknownVariables: false }`:
```

Comment:
`/* eslint css/no-invalid-properties: ["error", { allowUnknownVariables: false }] */`

in this example also.

Reply:
Done